### PR TITLE
Bug 1273779 - [TV][Home] Scrolling fails in the Card Picker mode

### PR DESCRIPTION
**Pull request by Fischer-L**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=400cc3d49435b3c72f8b04c173efceb1f835f1a2

File: tv_apps/smart-home/js/card_picker.js [hunk 1]
```diff
@@ -233,8 +233,8 @@
     },
 
     _scrollTo: function(elem) {
-      var scrollY = elem.offsetTop -
-              (this.gridView.offsetHeight - elem.offsetHeight) / 2;
+      var scrollY = elem.parentNode.offsetTop -
```
Comment (luke-chang) [inline tv_apps/smart-home/js/card_picker.js]: I'd prefer calling `_scrollTo(elem.parentNode)` instead in order to make this function name more meaningful.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=29683eea51dc890627282a28e498e24bd40d0457

File: tv_apps/smart-home/js/card_picker.js (hunk 1 above)
Comment (Fischer-L) [inline tv_apps/smart-home/js/card_picker.js]: OK, Updated, thanks

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=498f0e64c8689cd1170d1684656642828f15a494

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5fd430518ddbd1809033cfac2e7b498926093aa2

**Pull request merged**